src/core/buffers/representation/representation_buffer.ts

```diff
@@ -259,6 +260,14 @@ export default function RepresentationBuffer<T>({
                            ...neededSegments ];
       }
 
+      if (indexSegment !== null) {
+        // prepend index segment
+        const indexSegmentPriority = getSegmentPriority(indexSegment, timing);
```

Comment:
Currently, the segment priority is always set at 0 (because segment time is 0).
As i also consider that it is logical for index and init segments to be prioritary, i can set it direclty to 0 without calculating it.
It will not change the player behavior, and will be cleaner.

Reply:
Ok with me